src/firefly/html/demo/ffapi-highlevel-test.html

```diff
@@ -231,9 +231,12 @@
             firefly.addXYPlot({QUERY_ID: 'oldSelectable'}, 'oldXyPlot1');
             //-----< old xyplot api ------//
 
-            var regionAry = [
+            window.regionAry = [
+                 /*
                 'global color=green dashlist=8 3 width=1 font="helvetica 10 normal normal" select=1 highlite=1 dash=0 fixed=0 edit=1 move=1 delete=1 include=1 rotate=0 source=1',
                 'J2000;point    202.41   47.262 # color=pink text="pt circle 1" point=circle',
+                'box (100p, 70p, 20p, 20p, 0) #color=red text={IMAGE coordinate - 1}',
+                'box (100i, 70i, 30i, 30i, 0) #color=orange text={PHYSICAL coordinate -2}',
```

Comment:
IMAGE and PHYSICAL in the text should be switched for these two items: p is used for physical pixels, i - for image pixels